simulator/include/DatasetParser.h

```diff
@@ -0,0 +1,23 @@
+#ifndef CHS_DATASETPARSER_H
+#define CHS_DATASETPARSER_H
+
+#include <vector>
+
+#include "DatafileController.h"
+
+namespace CacheHierarchySimulator
+{
+
+/**
+ * Class for parsing dataset files into instructions that the simulator can process
+ */
+class DatasetParser
```

Comment:
If we are using a class just for one static function, it may be better to just declare a namespace and use an static extern function.

Reply:
agreed. I removed the class declaration and just left it as a function. I also updated the parser to use the ```Instruction``` datatype defined in ```api.h```